TicTacToe/Move.cs

```diff
@@ -0,0 +1,26 @@
+namespace TicTacToe
+{
+    public class Move
+    {
+        private readonly int _coordX;
+        private readonly int _coordY;
+
+        public Move(int coordX, int coordY)
+        {
+            _coordX = coordX;
+            _coordY = coordY;
+        }
+
+        public (int, int) GetCoord()
+        {
+            return (_coordX, _coordY);
```

Comment:
I think this would be better if you made the _coordX and _coordY Properties instead of them just being fields, they could have a public getter and then you wouldn't need GetCoord() and you probably wouldn't need to override the Equals because you can more easily check if the property values are equal to something else.